tools/tidb-binlog-kafka.md

```diff
@@ -89,9 +89,23 @@ Kafka 集群用来存储由 Pump 写入的 binlog 数据，并提供给 Drainer
     dir = "/path/pb-dir"
     ```
    
-*   Kafka 和 Zookeeper 集群的安装和配置不在本文赘述，需要在部署 TiDB-Binlog 之前部署好。Kafka 需要0.9及以上版本，且保证设置参数auto.create.topics.enable=true, 推荐部署在3～5台服务器上，机器的磁盘空间和具体的业务数据量相关。
+#### Kafka 和 Zookeeper 集群需要在部署 TiDB-Binlog 之前部署好。Kafka 需要0.9及以上版本。
+
+#### kafka 集群配置推荐 
 
+|名字|数量|内存|CPU|硬盘|
+|:---:|:---:|:---:|:---:|:---:|
+|Kafka|3+|16G|8+|2+ 1TB|
+|Zookeeper|3+|8G|4+|2+ 300G|
 
+#### kafka 配置参数推荐
+    
+ - auto.create.topics.enable=true 如果还没有创建topic，kafka会在broker上自动创建topic
```

Comment:
Add a whitespace between the Chinese and English characters. Please modify similar items involved in this PR.